compiler/testData/codegen/bytecodeText/forLoop/forInStringSpecialized.kt

```diff
@@ -9,5 +9,3 @@ fun test() {
 // 0 hasNext
 // 0 nextChar
 // 0 INVOKEINTERFACE
-// 1 ISTORE 4
```

Comment:
I opted for removing these. I'm not sure why we would want to check that we store and load integers in local 4? With this optimization we don't anymore. What was in local 4 was the constant 1 for the step, and we just use const_1 for that when needed now.

Reply:
Maybe @dnpetrov remember an aim for this.
@madsager Could we add // 1 charAt (I)C and // 1 length ()I ?

Reply:
Thanks! Added charAt and length lines. Looks like the matching doesn't like having the signature there though, so I had to leave that out.

Reply:
I believe it's a regex so you'd have to escape the parenthesis.

Reply:
Yes, it should be escaped \( \)

Reply:
Awesome. Thanks both! :)